src/cast_core/test/CrudActorYamlTests.yml

```diff
@@ -572,3 +572,21 @@ Tests:
     OutcomeCounts:
       - Filter: {a: 1}
         Count: 1
+
```

Comment:
Oof you're right I totally overlooked this as a workload yaml last time, my bad. This looks great!